88.md

```diff
@@ -0,0 +1,130 @@
+NIP-88
+======
+
+Discreet Log Contracts over nostr
+-----------------
+
+`draft` `optional`
+
+This NIP describes a way to use Discreet Log Contracts (DLCs) over Nostr. This comes in four main
+components: `kind:28_888` for sending DLC messages between wallets. `kind:88` for posting DLC offers to the global
+market. `kind:89` and `kind:90` for publishing DLC oracle announcements and attestations. And `kind:91` for a DLC
+"receipt" that can publish the result of a DLC for social interaction.
+
+## DLC Messages
+
+DLCs typically use [BOLT 8](https://github.com/lightning/bolts/blob/master/08-transport.md) for sending messages but
+this has various problems: it requires the ability to accept incoming connections and an always-on node. Nostr provides
+a unique opportunity to use a different transport layer for DLCs. This NIP proposes a new `kind:28_888` message that
+acts like a mailbox for DLC messages. The DLC messages are sent over nostr and available for the user the next time
+they open their wallet. This allows for a more mobile-friendly DLC experience.
+
+### `kind:28_888`
+
+// todo probably don't want an ephemeral event for this, but it's a good starting point
+
+Kind 28_888 is a simple message that contains a [NIP04](04.md) encrypted DLC message and it tagged with the recipient's
+public key with a `p` tag and if it is in reply to another event, that event should be tagged with an `e` tag.
```

Comment:
How should a client react to two competing messages following the same preceding event? 
I assume the client should take the last seen?

Reply:
Yeah a client should never send 2 messages and the receiver should only process the ones they expect.

Will add clarifications